assets/js/src/core/modules/data-object/hooks/use-data-object-helper.ts

```diff
@@ -121,5 +128,37 @@ export const useDataObjectHelper = (): UseDataObjectReturn => {
     }
   }
 
-  return { openDataObject, executeDataObjectTask }
+  const formatPath = async (items: IFormatPathItems[], fieldName: string, dataObjectId: number): Promise<DataObjectFormatPathApiResponse> => {
+    const targets = items.reduce((acc, item) => {
+      acc[`object_${item.id}`] = {
+        id: item.id,
+        type: item.type,
+        label: item.fullPath,
+        path: item.fullPath,
+        nicePathKey: `object_${item.id}`
+      }
+      return acc
+    }, {})
+
+    const { data } = await store.dispatch(api.endpoints.dataObjectFormatPath.initiate({
```

Comment:
Can we add some kind of caching here so that we don't request items with the same context information again after we already requested it? (so just remember the result in a local ref or so)